src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if (APPLE)
+   find_library(LIBGSS NAMES GSS)
+   add_definitions(-DGSSFW)
```

Comment:
Stands for "framework" since Apple documentation refers to it as "GSS framework"